test/controllers/api/docs_controller_test.exs

```diff
@@ -58,16 +52,59 @@ defmodule HexWeb.API.DocsControllerTest do
     assert conn.resp_body == "HEYO"
   end
 
+  @tag :integration
```

Comment:
Integration tag is needed since the test will hit S3 if the env vars for it are set.